{
  "name": "try-call",
  "version": "0.0.2",
  "description": "Functional try-catch",
  "main": "index.js",
  "scripts": {
    "test": "node test"
  },
  "keywords": [
    "try",
    "catch",
    "optimization"
  ],
  "repository": {
    "url": "git+ssh://git@github.com/n-johnson/try-call.git",
    "type": "git"
  },
  "author": {
    "name": "azer"
  },
  "license": "BSD",
  "gitHead": "22ad802c3e3f4898a5e388e120e611c45e08de86",
  "bugs": {
    "url": "https://github.com/n-johnson/try-call/issues"
  },
  "homepage": "https://github.com/n-johnson/try-call#readme",
  "_id": "try-call@0.0.2",
  "_shasum": "6e7dd18ddda109be795027b248d520e41a5d5145",
  "_from": "try-call@0.0.2",
  "_npmVersion": "3.7.2",
  "_nodeVersion": "4.3.0",
  "_npmUser": {
    "name": "nj48",
    "email": "nj48001@gmail.com"
  },
  "dist": {
    "shasum": "6e7dd18ddda109be795027b248d520e41a5d5145",
    "tarball": "https://registry.npmjs.org/try-call/-/try-call-0.0.2.tgz"
  },
  "maintainers": [
    {
      "name": "nj48",
      "email": "nj48001@gmail.com"
    }
  ],
  "directories": {},
  "_resolved": "https://registry.npmjs.org/try-call/-/try-call-0.0.2.tgz",
  "readme": "ERROR: No README data found!"
}
